### DISCUSSION TITLE
Php 7?

### DISCUSSION THREAD

**Discussion by fossildj** [nutgram/nutgram#345] [Q&A] [answered]
Can I run nutgram on php 7.3? I suspect is the ForceReply:: giving error

I encounter error on my bot

26-Feb-2023 09:31:12 UTC] PHP Parse error:  syntax error, unexpected ':', expecting ')' in /home/mcsgarag/public_html/ben/bots/panorama_bot/panorama.php on line 38

This is my line 38

      ```
$bot->sendMessage(UNAUTHORISED_USER, [
                'reply_markup' => ForceReply::make(
                    force_reply: true,
                    input_field_placeholder: 'c****w',
                    selective: true,
                ),
                'parse_mode' => 'html'
        ]);
```

**Comment by Lukasss93** [comment, marked as answer]
No. You need PHP >=8.0

**Reply by fossildj** [reply]
Thanks
